source/includes/fundamentals/code-snippets/Query.java

```diff
@@ -50,36 +50,51 @@ public static void main(String [] args){
     }
 
     private void comparisonFilter(){
+        // Creates a filter to match documents that have a "qty" value over 7
         // begin comparisonFilter
```

Comment:
```suggestion
        // begin comparisonFilter
```

Reply:
Are these for includes reasons? If so, ignore the suggestion. If not, please remove.

Reply:
Yes, these are to mark the code to include